tests/test_breadth_first_search.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from unittest.mock import Mock, call
+
+from src.breadth_first_search import breadth_first_search
+
+
+class TestBreadthFirstSearch(unittest.TestCase):
+    def setUp(self):
+        self.fixture = {
+            "first": ["second", "third"],
+            "second": ["third", "fourth", "me-1", "sixth"],
+            "third": ["first", "fifth"],
+            "fourth": ["me-2"],
+            "fifth": ["second", "seventh"],
+            "sixth": [],
+            "seventh": [],
+            "me-1": [],
+            "me-2": []
+        }
```

Comment:
if you decide that node names is always right, remove empty nodes

Reply:
ok